common/test.c

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <debug.h>
+#include <test.h>
+
+#if TEST
```

Comment:
Please add a blank line after the preprocessor directive

Reply:
done